### ISSUE TITLE
Rename app/Resources/views/templates

### ISSUE THREAD

**Issue by sagikazarmark** [sulu/sulu-minimal#18]
The naming of this directory is a little bit inconsistent: it contains templates for app/Resources/templates/pages

Is it possible to bring the naming closer to each other?

**Comment by danrot** [comment]
Well, `app/Resources/views` is kind of given by Symfony. What would your suggestion be?

**Comment by sagikazarmark** [comment]
What I really meant is that views for pages stored in `app/Resources/templates/pages` are stored in `app/Resources/views/templates`.

Could it be `app/Resources/views/pages`?

Though I am using sulu for a month in a prod app now, it's still confusing me all the time.

**Comment by danrot** [comment]
Hm, I like that idea... Let me just mention a few people here, to ask for their opinions: @webmozart @chirimoya @wachterjohannes @alexander-schranz

**Comment by wachterjohannes** [comment]
sounds reasonable and more practicable

**Comment by alexander-schranz** [comment]
following folder structure is used by use under views:

```
 - templates
    - articles
      - article.html.twig
    - pages
      - homepage.html.twig
      - default.html.twig
    - ...
  - master.html.twig
```

but until you don't have additional types like Articles of templates I personally would omit the `pages` folder /cc @sulu/core-team

**Comment by danrot** [comment]
Isn't that something you have touched in the PR for the Symfony 4 folder structure? Can you maybe link them here correctly?

**Comment by danrot** [comment]
Fixed in #98 :tada:
